client/server/pages/index.js

```diff
@@ -832,6 +832,12 @@ function wpcomPages( app ) {
 		const redirectUrl = '/setup/domain-transfer';
 		res.redirect( 301, redirectUrl );
 	} );
+
+	// Redirects from /help/courses to https://wordpress.com/learn/courses.
+	app.get( '/help/courses', function ( req, res ) {
+		const redirectUrl = 'https://wordpress.com/learn/courses';
```

Comment:
I've just tested this and I think we should use a relative URL instead so we stay within the current environment.
```suggestion
		const redirectUrl = '/learn/courses';
```